src/mvAppItem.cpp

```diff
@@ -2112,6 +2150,7 @@ DearPyGui::GetEntityParser(mvAppItemType type)
         args.push_back({ mvPyDataType::Bool, "horizontal", mvArgType::KEYWORD_ARG, "False", "Forces child widgets to be added in a horizontal layout." });
         args.push_back({ mvPyDataType::Float, "horizontal_spacing", mvArgType::KEYWORD_ARG, "-1", "Spacing for the horizontal layout." });
         args.push_back({ mvPyDataType::Float, "xoffset", mvArgType::KEYWORD_ARG, "0.0", "Offset from containing window x item location within group." });
+        args.push_back({ mvPyDataType::Bool, "disabled", mvArgType::KEYWORD_ARG, "False", "Disable everything inside the group. (Use mvThemeCol_TextDisabled and mvStyleVar_DisabledAlpha to edit the style of disabled widgets)" });
```

Comment:
I'd suggest that we use the standard `enabled` keyword instead of adding `disabled` - this way it will be more consistent with the rest of the API.

Also, regarding the help text for `disabled`: `mvThemeCol_TextDisabled` is not used by many widgets in the disabled state; they use `mvThemeCol_Text` instead. Some even use `mvThemeCol_TextDisabled` for other stuff, like `input_text` renders the hint with that color. Add to this that in DPG, "enabled" and "disabled" styles are configured separately via theme components with different `enabled_state`.

I'd suggest that we leave only `mvStyleVar_DisabledAlpha` in this comment - and maybe explain that it is applied to **all** colors within the disabled group.

Reply:
Is it really the same? Does `enable` parameter call `ImGui::BeginDisabled();`? Because that was the reason I put `disabled` keyword

Reply:
> Does enable parameter call ImGui::BeginDisabled();

No. What I suggest is that instead of adding disabled to the parser, add `MV_PARSER_ARG_ENABLED` to the flags on `case mvAppItemType::mvGroup`. Then, in `DearPyGui::draw_group`, call `ImGui::BeginDisabled()` if `item.config.enabled` is false (and you won't need `config.disabled` in this case).

Reply:
Oh okay, it makes sense, I'll do it!